### PR TITLE
Solve #316 - Add option to replace `in` used in `always_for_in`

### PR DESCRIPTION
**Pull request by domluna**
Add an option `for_in_replacement`, that works alongside `always_for_in` whereby you can specify an alternative to `in`.

Can be used when `always_for_in` is `true` to replace `in` with ∈ (\\in), or `=` instead:
```julia
for a = 1:10
end
# formatted with always_for_in = true, for_in_replacement = "∈"
for a ∈ 1:10
end
```

### CODE REVIEW

File: src/options.jl [hunk 1]
```diff
@@ -18,4 +18,12 @@ Base.@kwdef struct Options
     align_pair_arrow::Bool = false
     conditional_to_if::Bool = false
     normalize_line_endings::String = "auto"
+    for_in_replacement::String = "in"
+end
+
+function valid_for_in_op(s::String)
```
Comment (iblis17) [inline src/options.jl:24]:
```suggestion
valid_for_in_op(s::String) = s ∈ ("in", "=", "∈")
```

And... maybe we can run this function in constructor of `Options`?

Comment (domluna) [reply src/options.jl:24]: that makes a lot of sense

**Comment by domluna** [comment]
> Hi all, I'd really love to have this merged&released, is there anything I could potentially help with to get this through?

Probably just has to be rebased. And also do this https://github.com/domluna/JuliaFormatter.jl/pull/380#discussion_r650462183

**Comment by exaexa** [comment]
OK, I'll push it to my fork ASAP. The rebase will be slightly challenging it seems.

**Pull request closed without merging**